po/nl.po

```diff
@@ -819,6 +819,11 @@ msgstr "Maak een beleid"
 msgid "Create a new Event Handler"
 msgstr "Maak een nieuwe event handler"
 
+#: privacyidea/static/components/config/views/config.resolvers.http.html:5
+#, fuzzy
+msgid "Create a new HTTP Resolver"
+msgstr "Maak een nieuwe LDAP resolver"
```

Comment:
s/LDAP/HTTP/

Reply:
Your dutch is perfect! ;-)
I think these translations were "autoadded" by best match.
The NL translation was done *before* the HTTP resolver was merged.
So please ignore dutch. I will ask the translator from the community to update the durch translation.